login.css

```
@@ -0,0 +1,94 @@
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+header {
+  margin-top: 231px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+form,
+label {
+  width: 640px;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+label {
+  font-weight: 700;
+}
+
+input {
+  height: 36px;
+  background-color: var(--gray100);
+  border: 0;
+  border-radius: 12px;
+  padding: 10px 24px;
+  font-size: 16px;
+  flex-grow: 1;
+}
+
+form button {
+  padding: 16px 0px;
+  border: 0;
+  border-radius: 40px;
+  background-color: var(--primary-color);
+  color: #fff;
+  font-size: 20px;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+#password {
+  display: inline-flex;
+  position: relative;
+}
+
+#password>button {
+  position: absolute;
+  background-color: transparent;
+  right: 20px;
+  top: 0px;
+  z-index: 1;
+}
+
+.easy-login-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #e6f2ff;
+  border-radius: 8px;
+  padding: 16px 23px;
+  font-size: 16px;
+  font-weight: 500;
+}
+
+.easy-login-icon {
+  display: flex;
+  gap: 16px;
+}
+
+.easy-login-icon>button {
```

포맷팅도 항상 일관적으로 적용될 수 있게끔 신경써주세요! 

```suggestion
.easy-login-icon > button {
```

---

아 이 부분은 제가 IDE를 바꾸면서 그랬던 것 같습니다! 앞으로는 이 컨벤션으로 갈 것 같아요!